bot/seasons/valentines/be_my_valentine.py

```diff
@@ -0,0 +1,232 @@
+import logging
+import random
+import typing
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+from discord.ext.commands.cooldowns import BucketType
+
+from bot.constants import Colours, Lovefest, Roles
+from bot.decorators import with_role
+
+log = logging.getLogger(__name__)
+
+HEART_EMOJIS = [":heart:", ":gift_heart:", ":revolving_hearts:", ":sparkling_heart:", ":two_hearts:"]
+USER_LOVEFEST = []
+
+
+class BeMyValentine:
+    """
+    A cog that sends valentines to other users !
+    """
+    id = Lovefest.role_id
+
+    def __init__(self, bot, valentines):
+        self.bot = bot
+        self.valentines = valentines
+
+    @commands.group(name="lovefest", invoke_without_command=True)
+    async def add_role(self, ctx):
```

Comment:
This needs to be its own command within the group, otherwise it's not going to be clear how to invoke the command (or what command is being invoked):

![image](https://user-images.githubusercontent.com/5323929/52885226-8922bf80-313e-11e9-901e-b2c919b360a2.png)

Reply:
or lemme just make 2 seperate commands insted of a group?